portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +29,48 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+/**
+ * Adds a photo grid with different width/height options to the page
+ */
+
+// Get the elements with class="column"
+var elements = document.getElementsByClassName("column");
+
+// Declare a loop variable
+var i;
```

Comment:
For readability and error prevention, I would remove this and just add var to each loop. There is no major efficiency gain by having a global var i being reused.

for (var i = 0;....)